mistletoe/contrib/mathjax.py

```diff
@@ -18,11 +18,11 @@ def __init__(self, **kwargs):
 
     def render_math(self, token):
         """
-        Ensure Math tokens are all enclosed in two dollar signs.
+        Convert single dollar sign enclosed expressions to \(\) inline math delimiter for mathjax
         """
-        if token.content.startswith('$$'):
+        if token.content.startswith('$$') or token.content.startswith('\\[') or token.content.startswith('\\('):
```

Comment:
Not sure about the extended condition here, because `Math` token should only be parsed from text enclosed in `$$` or `$` in mistletoe:

```python
class Math(span_token.SpanToken):
    pattern = re.compile(r'(\${1,2})([^$]+?)\1')
    parse_inner = False
    parse_group = 0
```

So the extended condition would make sense if we also extended the existing `pattern`. But do we want it?

Reply:
I didnt want to mess with the parsing, only the rendering, and erroneously assumed they were supported so i added them in there. it is a question of mistletoe and markdown as a standard if it should support ``\( \) \[ \]`` and if that will cause other problems like some collision in syntax(i am not sure). 

that being said, i think githubs markdown rendering supports them, and they are both mathjax and latex syntax, so tokenizing them as math with the properly modified regex is all it takes to make them render cleanly in both.

Reply:
\[ 
\sum\limits_{x=1}^{5}x^2
\]

inline \(\int x^3\)

Reply:
OK, then I think we can just leave the original `if token.content.startswith('$$'):` condition in here.

Reply:
Done - we don't extend the parsed Math syntax for now.